requirements/zstd.txt

```diff
@@ -1 +1 @@
-zstandard
+backports.zstd>=1.0.0;python_version<'3.14'
```

Comment:
I would prefer we keep using zstandard to limit the scope of these changes.

Reply:
@Rogdham what are the benefits to using backports.zstd vs zstandard? I'd prefer to avoid changing our dependency unless there is a strong motivation.